codelists/codelists.txt

```diff
@@ -28,3 +28,4 @@ opensafely/solid-organ-transplantation/2020-04-10
 opensafely/permanent-immunosuppression/2020-06-02
 opensafely/temporary-immunosuppression/2020-04-24
 user/peter-inglesby/eth2001/0f56fa3b
+primis-covid19-vacc-uptake/learndis/v1
```

Comment:
The url slug confused me here as it doesn't match the name of codelist. I have double checked and it is correct.